server.go

```diff
@@ -1590,10 +1592,16 @@ func (s *server) Stop() error {
 		s.channelNotifier.Stop()
 		s.peerNotifier.Stop()
 		s.htlcNotifier.Stop()
-		s.cc.Wallet.Shutdown()
-		s.cc.ChainView.Stop()
+		if err := s.cc.Wallet.Shutdown(); err != nil {
+			srvrLog.Warnf("unable to stop Wallet: %v", err)
```

Comment:
nit: logs should be capitalized.

Reply:
I only changed the ones that I changed in this PR. There are some other lowercase logs in the same function.